apps/admin-portal/src/store/actions/authenticate.ts

```diff
@@ -131,47 +129,42 @@ export const getProfileInformation = () => (dispatch): void => {
         });
 };
 
-/**
- * Initialize identityManager client
- */
-const identityManager = (() => {
-    let instance: ConfigInterface;
-
-    const createInstance = () => {
-        return new IdentityClient({
-            callbackURL: store.getState().config.deployment.loginCallbackUrl,
-            clientHost: store.getState().config.deployment.clientHost,
-            clientID: store.getState().config.deployment.clientID,
-            responseMode: process.env.NODE_ENV === "production" ? "form_post" : null,
-            scope: [ TokenConstants.SYSTEM_SCOPE ],
-            serverOrigin: store.getState().config.deployment.serverOrigin,
-            tenant: store.getState().config.deployment.tenant,
-            tenantPath: store.getState().config.deployment.tenantPath
-        });
-    };
-
-    return {
-        getInstance: () => {
-            if (!instance) {
-                instance = createInstance();
-            }
-
-            return instance;
-        }
-    };
-})();
-
 /**
  * Handle user sign-in
  */
 export const handleSignIn = () => (dispatch) => {
-    identityManager.getInstance().signIn(
-        () => {
-            dispatch(setSignIn());
-            dispatch(getProfileInformation());
+    const oAuth = OAuth.getInstance();
+    oAuth
+        .initialize({
```

Comment:
How do we handle the response mode?

Reply:
Added response mode to all initialize methods.